### PR TITLE
[improve] Upgrade wildfly-eytron (used by debezium) to fix CVE-2022-3143

### PR DESCRIPTION
**Pull request by dlg99**
### Motivation

OWASP detects CVE-2022-3143

### Modifications

Upgraded wildfly-eytron (used by debezium) to fix CVE-2022-3143
Upgraded OWASP checker plugin (mainly to pick up version that uses hosted suppressions file to deal with well known false positives).

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If the box was checked, please highlight the changes*

- [x] **Dependencies (add or upgrade a dependency)**
- [ ] The public API
- [ ] The schema
- [ ] The default values of configurations
- [ ] The threading model
- [ ] The binary protocol
- [ ] The REST endpoints
- [ ] The admin CLI options
- [ ] The metrics
- [ ] Anything that affects deployment

### Documentation

- [ ] `doc` 
- [ ] `doc-required` 
- [x] `doc-not-needed` 
- [ ] `doc-complete` 

### Matching PR in forked repository

PR in forked repository: https://github.com/dlg99/pulsar/pull/9

### CODE REVIEW

File: pulsar-io/debezium/oracle/pom.xml [hunk 1]
```diff
@@ -31,6 +31,41 @@
   <name>Pulsar IO :: Debezium :: oracle</name>
 
   <dependencies>
+    <dependency>
```
Comment (nicoloboschi) [inline pulsar-io/debezium/oracle/pom.xml:34]: we should add those in the <dependencyManagement> section

Comment (dlg99) [reply pulsar-io/debezium/oracle/pom.xml:34]: didn't get it, in what section?

**Review by dave2wave — APPROVED** [review]
We need to push through these types of upgrades. If our testing framework isn;t sufficient then upgrade that!

File: pulsar-io/debezium/oracle/pom.xml (hunk 1 above)
Comment (lhotari) [reply pulsar-io/debezium/oracle/pom.xml:34]: I think @nicoloboschi means that these should be in dependencyManagement section. I agree. 
Please define these in a dependencyManagement section of `pulsar-io/debezium/pom.xml` file.

**Review by lhotari — CHANGES_REQUESTED** [review]

**Review by lhotari — APPROVED** [review]

**Pull request merged**